openms-wiki/contributor-faq.md

```diff
@@ -0,0 +1,269 @@
+Contributor FAQ
```

Comment:
Let's add this file as:

`OpenMS-docs/docs/faqs/contributor-faq.md`.

Reply:
done